utils/src/test/resources/log4j.properties

```diff
@@ -14,8 +14,38 @@
 #
 
 # Set everything to be logged to the console
-log4j.rootCategory=WARN, console
-log4j.appender.console=org.apache.log4j.ConsoleAppender
-log4j.appender.console.target=System.err
-log4j.appender.console.layout=org.apache.log4j.PatternLayout
-log4j.appender.console.layout.ConversionPattern=%d{yy/MM/dd HH:mm:ss} %p %c{1}: %m%n
+log4j.rootCategory = WARN, general
+
+log4j.appender.general = org.apache.log4j.ConsoleAppender
+log4j.appender.general.target = System.err
+log4j.appender.general.layout = org.apache.log4j.PatternLayout
+log4j.appender.general.layout.ConversionPattern = [%p] %d{yy/MM/dd HH:mm:ss} %c{1}: %m%n
+
+# Suppress warnings logged within the code inside the tests
+log4j.logger.za.co.absa.enceladus.conformance.interpreter.OptimizerTimeTracker = ERROR
```

Comment:
My personal preference is to keep `OptimizerTimeTracker` warnings. It may help track timing issues in case it occurs (which they shouldn't). But I don't insist on this.